nuklear.h

```diff
@@ -373,8 +373,10 @@ extern "C" {
     #elif (defined(_WIN32) || defined(WIN32)) && defined(_MSC_VER)
       #define NK_SIZE_TYPE unsigned __int32
     #elif defined(__GNUC__) || defined(__clang__)
-      #if defined(__x86_64__) || defined(__ppc64__) || defined(__PPC64__) || defined(__aarch64__)
+      #ifdef __LP64__
```

Comment:
Rather than replacing the compiler checks, could we append it to the OR list instead? For example....
``` c
#if defined(__x86_64__) || defined(__ppc64__) || defined(__PPC64__) || defined(__aarch64__) || defined(__LP64__)
```

I'd be worried about older infrastructures that define one of the above, but skip `__LP64__`. Want to keep backward compatibility.